spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-config-it/src/test/java/org/springframework/cloud/kubernetes/client/config/it/ConfigMapAndSecretIT.java

```diff
@@ -155,21 +120,56 @@ public void testConfigMapAndSecretWatchRefresh() throws Exception {
 	}
 
 	@Test
-	public void testConfigMapAndSecretPollingRefresh() throws Exception {
+	void testConfigMapAndSecretPollingRefresh() throws Exception {
 		deployConfigK8sClientPollingIt();
 
 		// Check to make sure the controller deployment is ready
 		k8SUtils.waitForDeployment(SPRING_CLOUD_CLIENT_CONFIG_IT_DEPLOYMENT_NAME, NAMESPACE);
 		testConfigMapAndSecretRefresh();
 	}
 
+	void testConfigMapAndSecretRefresh() throws Exception {
+
+		String propertyURL = "localhost:" + K3S.getMappedPort(80) + "/myProperty";
```

Comment:
I have slightly changed the code in this test, by using `WebClient`, or waiting for ingress, etc. Semantically, it checks the same things